.github/workflows/test_ropenscpca.yml

```
@@ -0,0 +1,39 @@
+on:
+  pull_request:
+    branches:
+      - main
+      - feature/*
+
+name: Check the rOpenScPCA package
+
+jobs:
+  R-CMD-check-renv:
+    runs-on: ubuntu-22.04
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Set up R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: 4.4.0
+          use-public-rspm: true
+
+      - name: Remove `.Rprofile` to not activate renv
+        run: |
+          rm ./packages/rOpenScPCA/.Rprofile
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: |
```

I think you need to specify the directory here. (And you should not need to specify `testthat` as that is part of the DESCRIPTION)
```suggestion
        with:
          working-directory: "packages/rOpenScPCA/"
          extra-packages: |
```